packages/utils/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@hitachivantara/uikit-react-utils",
+  "version": "0.1.0",
```

Comment:
Version bellow 1.0 is intentional? It means consumers should consider the exposed APIs as unstable?